README.md

```diff
@@ -113,6 +113,8 @@ and flags are:
       (default Info)
   -logprefix string
     	Prefix to log lines before logged messages (default "> ")
+  - maxpayloadsizekb
+       MaxPayloadSizeKb is the maximum size of payload to be generated by the EchoHandler size= argument (In kilobytes defaults to 256)
```

Comment:
manually wrap to 80 cols (I need to find a tool to do that) - when doing it pay attention to tab vs space, copy the prefix from another existing wrapped line